src/sas/sascalc/dataloader/readers/cansas_reader_HDF5.py

```diff
@@ -192,17 +192,17 @@ def read_children(self, data, parent_list):
                     self.current_dataset.mask = data_set.flatten()
                     continue
                 # Transmission Spectrum
-                elif key == u'T' and self.parent_class == \
-                        u'SAStransmission_spectrum':
+                elif (key == u'T') \
+                        and (self.parent_class == u'SAStransmission_spectrum'):
```

Comment:
You can wrap the entire condition in parentheses, then break before the operator:
```
              elif (key == u'T'
                      and self.parent_class == u'SAStransmission_spectrum'):
```